### ISSUE TITLE
Segmentation fault in cli mode on MacBook

### ISSUE THREAD

**Issue by albenik** [tarantool/tarantool#2954]
Tarantool was running on my dev MackBook from console to taranool cli and keep working while I used MakBook in regular way with multiple sleeps & wake ups.

Today I found that tarantool was crashed.

```
tarantool> 2017-11-23 22:19:54.058 [33383] main/105/checkpoint_daemon I> making snapshot...
2017-11-23 22:19:54.060 [33383] snapshot/101/main I> saving snapshot `./00000000000000000012.snap.inprogress'
2017-11-23 22:19:54.062 [33383] snapshot/101/main I> done
2017-11-23 22:19:54.063 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Thu Nov 23 23:19:54 2017
2017-11-23 23:19:54.069 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Fri Nov 24 00:19:54 2017
2017-11-24 00:19:54.068 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Fri Nov 24 01:19:54 2017
2017-11-24 01:49:09.471 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Fri Nov 24 02:49:09 2017
2017-11-24 02:49:09.478 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Fri Nov 24 03:49:09 2017
2017-11-24 03:49:09.488 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Fri Nov 24 04:49:09 2017
2017-11-24 04:49:09.494 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Fri Nov 24 05:49:09 2017
2017-11-24 05:49:09.501 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Fri Nov 24 06:49:09 2017
2017-11-24 06:49:09.503 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Fri Nov 24 07:49:09 2017
2017-11-24 07:49:09.505 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Fri Nov 24 08:49:09 2017
2017-11-24 08:49:09.512 [33383] main/105/checkpoint_daemon I> scheduled the next snapshot at Fri Nov 24 09:49:09 2017
Segmentation fault
Current time: 1511503052
Please file a bug at http://github.com/tarantool/tarantool/issues
Attempting backtrace... Note: since the server has already crashed, 
this may fail as well
#0  0x10dbf7d53 in print_backtrace+9
#1  0x10db5d238 in _ZL12sig_fatal_cbi+ad
#2  0x7fff62bb2f5a in _sigtramp+1a
#3  0x10dce806a in clear_pending+22
#4  0x10dce825a in ev_timer_again+12
#5  0x10dce7548 in ev_invoke_pending+5d
#6  0x10dce7b92 in ev_run+62b
#7  0x10dbe739b in tarantool_lua_run_script+93
#8  0x10db5ccaf in main+4ba
[1]    33383 abort      tarantool
```

```
  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro14,1
  Processor Name:	Intel Core i5
  Processor Speed:	2,3 GHz
  Number of Processors:	1
  Total Number of Cores:	2
  L2 Cache (per Core):	256 KB
  L3 Cache:	4 MB
  Memory:	8 GB
  Boot ROM Version:	MBP141.0167.B00

  System Version:	macOS 10.13.1 (17B48)
  Kernel Version:	Darwin 17.2.0
  Boot Volume:	Macintosh HD
  Boot Mode:	Normal
  Secure Virtual Memory:	Enabled
  System Integrity Protection:	Enabled
  Time since boot:	22 days 19:53
```

**Comment by kostja** [comment]
Thank you for your report.

**Comment by RunsFor** [comment]
Description is very similar to #2527

**Comment by albenik** [comment]
@RunsFor homebrew tarantool 1.7.5.184 bottle

**Comment by RunsFor** [comment]
@albenik This bug should be fixed in 1.7.6. Could you please try on the newer version? There is one available:

```bash
$ brew info tarantool                                                                                                                                       
tarantool: stable 1.7.6.12 (bottled), HEAD
In-memory database and Lua application server
```

To update use following commands:

```bash
$ brew update
$ brew upgrade tarantool
```

**Comment by albenik** [comment]
@RunsFor Confirm, there is no crashes anymore with new version
